submissions/phpslonuk/Popup/index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooli-style Popup</title>
+  <link rel="shortcut icon" type="image/png"
+    href="https://raw.githubusercontent.com/phpslonuk/Hooli-style_Popup/main/popup-icons/favicon.png" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <div class="logo"></div>
+      <div class="nav">
+        <ul class="nav_list">
+          <li class="nav_item nav_item_link">
+            <a href="#" class="nav_item_mail">Mail</a>
+          </li>
+          <li class="nav_item nav_item_link">
+            <a href="#" class="nav_item_images">Images</a>
+          </li>
+          <li class="nav_item wrapper">
+            <input type="checkbox" id="trigger" class="nav_item_checker" name="nav_item_checkbox">
+            <label class="popup_icon" for="trigger"> </label>
+            <div class="popup_menu">
+              <div class="popup-content">
+                <div class="main-menu">
```

Comment:
would be better to add the markup `nav > ul > li` instead of `div`  for "popup_menu". Could you please fix it?